internal/sbi/server.go

```diff
@@ -0,0 +1,199 @@
+package sbi
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"runtime/debug"
+	"sync"
+	"time"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+
+	"github.com/free5gc/openapi/models"
+	pcf_context "github.com/free5gc/pcf/internal/context"
+	"github.com/free5gc/pcf/internal/logger"
+	"github.com/free5gc/pcf/internal/sbi/consumer"
+	"github.com/free5gc/pcf/internal/sbi/processor"
+	"github.com/free5gc/pcf/internal/util"
+	"github.com/free5gc/pcf/pkg/app"
+	"github.com/free5gc/pcf/pkg/factory"
+	"github.com/free5gc/util/httpwrapper"
+	logger_util "github.com/free5gc/util/logger"
+)
+
+const (
+	CorsConfigMaxAge = 86400
+)
```

Comment:
remove

Reply:
@andy89923 or should it be moved to `free5gc/pcf/internal/sbi/api_oam.go`, where the cors headers are set?

Reply:
Sure!